modulepath-tests/pom.xml

```diff
@@ -50,39 +61,6 @@
                     <release>9</release>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration combine.self="override">
-                    <!--
-                    Allow access to Operating system metrics:
-                        open jdk.management/com.sun.management.internal
-
-                    Avoid warnings caused by reflection in
-                    SelectorOptimizer:
-                        open java.base/sun.nio.ch
-                    FilteringClassLoader:
-                        open java.base/java.lang
-                    TimedMemberStateFactoryHelper:
-                        open java.management/sun.management
-
-                    Powermock issue workaround (https://github.com/powermock/powermock/issues/905):
-                        export java.xml/jdk.xml.internal
-                     -->
-                    <argLine>
-                        ${vmHeapSettings}
-                        --add-opens jdk.management/com.sun.management.internal=com.hazelcast.core
-                        --add-opens java.base/sun.nio.ch=com.hazelcast.core
-                        --add-opens java.base/java.lang=com.hazelcast.core
-                        --add-opens java.management/sun.management=com.hazelcast.core
-                        --add-exports java.xml/jdk.xml.internal=com.hazelcast.core
-                        --illegal-access=warn
-                        -Dhazelcast.phone.home.enabled=false
-                        -Dlog4j.skipJansi=true
-                        ${extraVmArgs}
```

Comment:
Shouldn't we remove only the module args from here? I would also retain the comments.

Reply:
Nope, it's not necessary to keep this custom surefire plugin configuration. We reuse the config from the parent project instead (`hazelcast-root`). To workaround the `parallelTest` issue in surefire version 2.22.2 we disable `default-test` execution in modulepath-tests. So tests run always in the `singlejvm` execution.

Reply:
Thanks it's clear now :+1: